src/modules/dashboard/lists/BorrowedPositionsList/BorrowedPositionsListItem.tsx

```diff
@@ -83,7 +85,7 @@ export const BorrowedPositionsListItem = ({
           <Trans>Repay</Trans>
         </Button>
         <Button
-          disabled={!isActive || !borrowingEnabled || isFrozen}
+          disabled={!isActive || !borrowingEnabled || isFrozen || borrowCap.isMaxed}
```

Comment:
We should probably add a similar check to the `BorrowAssetsListItems` components

Reply:
I think this section was fixed but I don't changes to these files so l'll check on this

Reply:
The availableBorrows is calculated in `BorrowAssetsList` using the `getMaxAmountAvailableToBorrow` function, which factors in borrow cap and button is disabled if availableBorrows === 0, so this should be working correctly now